src/everest/optimizer/everest2ropt.py

```diff
@@ -416,9 +426,8 @@ def _parse_optimization(ever_config: EverestConfig, ropt_config):
         raise RuntimeError("Only one of 'options' and 'backend_options' allowed.")
     # The constraint_tolerance option is only used by Dakota:
     if backend == "dakota":
-        output_constraints = ever_config.output_constraints or None
         alg_const_tol = ever_opt.constraint_tolerance or None
-        if output_constraints is not None and alg_const_tol is not None:
+        if add_constraint_tolerance and alg_const_tol is not None:
```

Comment:
Nitpick: This variable says if there are output constraints specified? If that is the case, I would rename argument to something along the lines of `has_output_constraints`